pkg/types/clusterconfig/cluster_config.go

```diff
@@ -1243,6 +1251,9 @@ func (cc *CoreConfig) TelemetryEvent() map[string]interface{} {
 	if !strings.HasPrefix(cc.ImageAsyncGateway, "cortexlabs/") {
 		event["image_async_gateway._is_custom"] = true
 	}
+	if !strings.HasPrefix(cc.ImageEnqueuer, "cortexlabs/") {
```

Comment:
I can think of one other place in the codebase to add this image (`create.md`), and it would be worth doing a quick search to see if there are any others